source/delayed_capture/index.html.md.erb

```diff
@@ -18,27 +18,34 @@ You cannot use delayed capture for:
 * payments made with [payment links](https://www.payments.service.gov.uk/govuk-payment-pages/)
 * [recurring payments](/recurring_payments)
 
-To use this feature, include `"delayed_capture": true` in the body of a [Create new payment](/making_payments/#creating-a-payment) request.
+To use this feature, include `"delayed_capture": true` in the body of your [request to create a new payment](/making_payments/#creating-a-payment):
+
+`POST /v1/payments`
+
+```json
+{
+  "amount": 5000,
+  "reference" : "LE2504",
+  "description": "Pay a penalty charge notice (PCN)",
+  "return_url": "https://your.service.gov.uk/completed",
+  "delayed_capture": true
+}
+```
 
-The user experience matches the current payment flow. Once your user selects
-__Confirm__ on the __Confirm your details__ page:
+Your user will follow the standard payment flow. After your user selects __Confirm__ on the __Confirm your details__ page:
 
-- the [payment status](/api_reference/#payment-status-lifecycle) will change to `capturable`
-- your service can make a request to the
-`POST /v1/payments/{PAYMENT_ID}/capture` endpoint to send a delayed capture
-request
+- the [payment's status](/api_reference/#payment-status-lifecycle) will change to `capturable`
+- your service can make a request to the `POST /v1/payments/{PAYMENT_ID}/capture` endpoint to send a delayed capture request
 
-Your service must send the delayed capture request within 120 hours (5 days) of creating the payment, regardless of how long your user takes to complete the payment flow. If you do not, [the payment will expire](/payment_flow/#6-your-user-confirms-payment).
+Your service must send the delayed capture request within 120 hours (5 days) of creating the payment, regardless of when your user completes the payment flow. If you do not, [the payment will expire](/payment_flow/#6-your-user-confirms-payment).
```

Comment:
"Your service must send" becomes "If you do not" - should it be "if your service does not", rather than suddenly addressing an individual?

I am also wondering about the translation of a specific mention of the length of the user takes becoming "when". To me "how long" is a measure of time and "when" could be something that's not just specific to the length of time to complete. This may be overkill, though, so just my thoughts.

Reply:
I think you might be overthinking it but I understand what you're saying. This is very nitty gritty BUT I changed this because typically if a payment takes ages to be paid, it's not the user's fault.

What will typically happen is that a user will go to a government service (Apply for a passport, for example), go through their application process, then make a payment at the end of it. When it's time to pay, the service will send an API request to Pay, and the service will redirect the user to a URL we provide. It would be very rare for a user to fill in an application, get sent to the payment page, then just leave that payment page open for 5 days.

'... how long your user takes to complete the payment flow' sounds to me like the burden is on the user for not paying quickly enough, when typically that wouldn't be the case.

I really hope this makes sense.

Reply:
"I changed this because typically if a payment takes ages to be paid, it's not the user's fault." <--- aha!

That's fine, then. I did try to run through some scenarios for "when", but just didn't know enough about it to make a good picture.